docs/concepts/loss-functions.md

```diff
@@ -1,6 +1,164 @@
 (loss-function)=
 # {octicon}`number` Loss Functions
 
+The choice of loss functions relies heavily on your data. In summary, you should utilize loss functions when:
+
+| data type                                 | loss function                                     |
+|-------------------------------------------|---------------------------------------------------|
+| article-label tuple                       | `TripletMarginLoss`, `ArcFaceLoss`, `CosFaceLoss` |
+| text-image tuple                          | `CLIPLoss`                                        |
+| query-article-score triplets              | `CosineSimilarityLoss`                            |
+| query-article tuple                       | `MultipleNegativeRankingLoss`                     |
+| query-article-irrelevant_article triplets | `MarginMSELoss`                                   |
+
+For more information, please checkout {doc}`/concepts/data-preparation` .
+
+## TripletMarginLoss
+
+`TripletMarginLoss` is a *contrastive* loss function, meaning that the loss is calculated by comparing the embeddings of multiple documents (3 to be exact) documents to each other.
+Each triplet of documents consists of an anchor document, a positive document and a negative document.
+The anchor and the positive document belong to the same class, and the negative document belongs to a different class.
+The goal of `TripletMarginLoss` is to maximise the difference between the distance from the anchor to the positive document, and the distance from the anchor to the negative document.
+
+## ArcFaceLoss and CosFaceLoss
+
+SphereFace loss is a loss function that was first formulated for computer vision and face recognition tasks.
+Finetuner supports two variations of this loss function, `ArcFaceLoss` and `CosFaceLoss`.
+Instead of attempting to minimise the distance between positive pairs and maximise the distance between negative pairs, the SphereFace loss functions compare each sample with an estimate of the center point of each classes' embeddings.
```

Comment:
```suggestion
Instead of attempting to minimize the distance between positive pairs and maximize the distance between negative pairs, the SphereFace loss functions compare each sample with an estimate of the center point of each class's embeddings.
```